bin/_mkextrastep.bat

```diff
@@ -0,0 +1,61 @@
+:: Extra Step Terminal API
```

Comment:
> That one was going to be a solution of creating a category option for
assets.

Was going to? Is it not currently used? Then I think we should remove it from the PR.